### ISSUE TITLE
add function to predict out of sample values using best models

### ISSUE THREAD

**Issue by milandv** [snfagora/autotextclassifier#10]
Once a best model is found, we need to be able to apply the model to predict values (usually probabilities I think) for orgs not in the training or test set. (In fact, for an entire index of orgs.)

I think this should be particularly straightforward for embedding since each document will end up in the same vector space. 

When we did this before, this was a little trickier for n-grams since there were potentially n-grams in the training matrix that don't appear in any out of sample document. (And n-grams in out of sample documents that weren't in the training matrix but those can be dropped.)

**Comment by milandv** [comment]
Out of sample actually isn't the right name for this in the title of the issue. 

For the "does organizing" model, for instance, after building a model I pulled a file that had 100k random orgs that weren't part of building the model and predicts values (with probabilities) for them.

So what we need is just a function that takes new data + a model as an input and then applies the same recipe and predicts for the new data. 

I had to do this in 10k chunks because 100k orgs at once was too big a vector and crashed R.
